nubison_model/utils.py

```diff
@@ -0,0 +1,12 @@
+from contextlib import contextmanager
+from os import chdir, getcwd
+
+
+@contextmanager
+def temporary_cwd(new_dir):
```

Comment:
What's the reason for moving the temporary_cwd function from the test directory to the nubison_model directory?

Reply:
The same functionality is required to production code `Service.test_client()`